frontend/packages/kubevirt-plugin/src/selectors/vm/selectors.ts

```diff
@@ -155,3 +155,8 @@ export const getNodeSelector = (vm: VMGenericLikeEntityKind) =>
   vm.kind === VirtualMachineInstanceModel.kind
     ? (vm as VMIKind)?.spec?.nodeSelector
     : asVM(vm)?.spec?.template?.spec?.nodeSelector;
+
+export const getTolerations = (vm: VMGenericLikeEntityKind) =>
```

Comment:
on a second thought - we should split these

- better for separation of concern/refactoring and when one wants to use the simple selector only

- not saying we have to abandon this method, but it can call these two other callbacks

Reply:
why would one care if its simple or not? he needs the tolerations and he gets it without caring about his entity type. if you ask me its simplifying the dev life and we should keep it.

Reply:
I am not saying we should not have this function.
It just adds to a better readability to have two simple funcitons - for example if you navigate to vm-wrapper and navigat to getTolerations - you can see a simple logic and not care about the switching 

if you need a generic one - you can use this one and have the vm / vmi / template switching logic here